debug-ui/scripts/stats-server.ts

```diff
@@ -128,73 +128,85 @@ export class ManifestStatsServer {
     };
 
     this.ws.onmessage = async (message) => {
-      const fill: FillLogResult = JSON.parse(message.data.toString());
```

Comment:
Is the fill's feed now sending multiple fills in a message? If that is the case, should be handled at the source